requirements.txt

```diff
@@ -1,7 +0,0 @@
-nodeenv==1.3.3
-ipdb==0.13.2
-flake8==3.8.3
-pre-commit==1.15.1
-twine
-bumpversion
-wheel
```

Comment:
I think the idea was that `requirements.txt` would be synthesized from `Pipfile.lock` whenever it was updated. That way someone can install the dependencies with `pip` rather than requiring use of `pipenv`. That's more useful for a project where the installation process is to clone a git repo or fetch a released archive. For a library installed from PyPI it's less useful as these requirements won't be used by `pip` and developers can be expected to use `pipenv`.